chapter_06/lib/models/plan.dart

```diff
@@ -1,18 +1,17 @@
 import '../repositories/repository.dart';
-import 'package:flutter/foundation.dart';
 import './task.dart';
 
 class Plan {
   final int id;
-  String name = '';
+  String name;
```

Comment:
I thought this assignment might be redundant as we are assigning it in the constructor on line 9.